### ISSUE TITLE
how accurate the retification needed for sttr to work

### ISSUE THREAD

**Issue by ynjiun** [mli0603/stereo-transformer#48]
I tried to run the inference using a stereo images captured by my stereo camera using pretrained models. But none of them showing reasonable depth prediction. However, if I use Kitti dataset, then it looks good. This makes me think that this might means:
1. my stereo images retification is not good enough, if so, is there a way (or metric) to specify how good a rectification need to be to run sttr?
2. or sttr is only good for the dataset it was trained for and not so good to run arbitray stereo images set?

Thank you for sharing your insight in advanced.

**Comment by mli0603** [comment]
Hi @ynjiun 

Thanks for reaching out. I assumed you are using the KITTI pretrained model? The KITTI pretrained model is severely overfitted to the small dataset. For better generalization, please use the scene flow trained model. Would you mind giving that a try?
